data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,569 @@
+			`	"And I'm Meroot," adds the Hai woman. "I was on Newhome walking to my research lab when some men pulled me into an alley and zapped me with a taser. I woke up in the brig of a space ship."`
+			`	"My experience was similar. I was walking from my biolab to my house on Serpens. Some men jumped out from behind a dune and zapped me."`
+			`	Meroot continues, "They took us both here. I was able to struggle out of my handcuffs and sneak the keys from the guard. We escaped together, and we've been hiding for weeks. Please take us to <destination>!"`
+			choice
+				`	"Sure. Let's go."`
+				`	"No, you'll have to find another pilot."`
+					decline
+			`	The two women put on clothing that thoroughly covers their person. They lead you through a series of ducts and alleys back to the pad where your ship is parked. Just as your airlock opens there's a sharp zapping sound as someone shoots Meroot with a stun gun. You and Lucy duck and a couple other shots dissipate harmlessly on the bulkhead of your ship. You fire off a couple shots back and between the two of you you manage to drag Meroot through the airlock, seal it behind you and lift off. It appears you're being followed into orbit.`
+				launch
+	on complete
+		log `Rescued a pair of researchers: a Hai woman named Meroot and a human woman named Lucy. They were kidnapped by men who brought them to Canyon. Returned them to <destination> where they now live together. They don't know the identities of the kidnappers.`
+		payment 315000
+		"hai slave prereq" ++
+		dialog `Meroot and Lucy thank you for your help and give you <payment>. Lucy decides to live in Hai space with Meroot. "It's safer here," she explains, though you have your doubts. You ask, but the two have no idea who the men that kidnapped them were.`
+	npc
+		government "Bounty Hunter that Won't Enter Hai Space"
+		personality plunders disables unconstrained
+		fleet "Large Core Pirates" 1
+		fleet "Small Core Pirates" 1
+			
+
+mission "Hai Rescue: Gipeep Hint"
+	landing
+	name "Lucrative Transport"
+	description "You've heard there's some suspiciously lucrative transport jobs on <destination>."
+	source
+		near "Nihal" 4
+		government "Republic" "Syndicate"
+		attributes "spaceport"
+		not attributes "station"
+		not planet "Maelstrom"
+	destination Maelstrom
+	to offer
+		"combat rating" > 50
+		has "Hai Leaks Response 6: active"
+		has "event: slightly lost"
+		not "Hai Rescue: Gipeep: offered"
+		not "event: hai-human treaty signed"
+		random < 80
+	on offer
+		conversation
+			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <planet>. Of course, it could mean nothing, but it's also very suspicious for that particular world <planet> is not known for lucrative transport jobs and it could likely be contraband, or worse...`
```

Comment:
```suggestion
			`As you come in to land at the spaceport, you listen to the usual chatter of captains coming and going. One guy is bragging particularly odiously about some lucrative transport jobs he'd picked up on <planet>. Of course, it could mean nothing, but it's also very suspicious for that particular world. <planet> is not known for lucrative transport jobs and it could likely be contraband, or worse...`
```